pkg/ui/console/policy.go

```diff
@@ -43,15 +44,22 @@ func FilterPolicies(policyPath string, policies policy.Policies) (policy.Policie
 		}
 		return policy.Policies{p}, nil
 	}
-	// run hub detector
+	// run hub detector. We got here if we couldn't find the policy specified by the command argument in the configuration
 	p, found, err := policy.DetectPolicy(policyName, subPath)
 	if err != nil {
 		return nil, err
 	}
 	if found {
 		return policy.Policies{p}, nil
 	}
-	return nil, fmt.Errorf("no policy with name %s found in configuration or remote. Available in config: %s", policyName, policies.All())
+
+	configPolicies := policies.All()
+	if len(configPolicies) == 0 {
+		return nil, diag.FromError(fmt.Errorf("no valid policy with name %s found. If using a local policy directory, ensure the path is correct and the directory exists", policyName), diag.USER)
```

Comment:
I would use `%q` when referring to user inputs. Line 61 also.

Reply:
Thanks for the tip! Done in https://github.com/cloudquery/cloudquery/pull/697/commits/7353ee0fc13cc7165471437df302f0fff230927a (I didn't wrap the array as it wraps each individual element and doesn't look that nice).

Reply:
ty, array isn't user input anyway.